j2lint/linter/indenter/node.py

```diff
@@ -24,10 +23,12 @@ class Node:
     """
     statement = None
     tag = None
+    parent = None
     node_start = 0
     node_end = 0
     children = []
     expected_indent = 0
+    block_start_indent = 0
```

Comment:
So I guess this was to remove the global variable `BLOCK_START_INDENT`.

Why not use an `def __init__(self)` constructor for Node rather than adding a class variable? is the indentation really shared across all instance of the Node class?